soccer/src/soccer/radio/network_radio.hpp

```diff
@@ -33,42 +33,42 @@ class NetworkRadio : public Radio {
     void receive() override;
     void switch_team(bool blue) override;
 
-    struct RobotConnection {
-        boost::asio::ip::udp::endpoint endpoint;
-        RJ::Time last_received;
-    };
+    boost::asio::ip::udp::endpoint base_station_endpoint = boost::asio::ip::udp::endpoint(
+        boost::asio::ip::address::from_string(kBaseStationAddress), kBaseStationPort);
 
-    // Connections to the robots, indexed by robot ID.
-    std::vector<std::optional<RobotConnection>> connections_{};
+    boost::asio::ip::udp::endpoint robot_status_endpoint = boost::asio::ip::udp::endpoint(
+        boost::asio::ip::address::from_string("127.0.0.1"), kIncomingBaseStationDataPort);
+    boost::asio::ip::udp::endpoint alive_robots_endpoint = boost::asio::ip::udp::endpoint(
+        boost::asio::ip::address::from_string("127.0.0.1"), kIncomingBaseStationAliveRobotsPort);
 
-    // Map from IP address to robot ID.
-    std::map<boost::asio::ip::udp::endpoint, int> robot_ip_map_{};
+    // Fuck Around and Find Out
+    // https://stackoverflow.com/questions/26243008/error-initializing-a-boost-udp-socket-with-a-boost-io-service
+    boost::asio::io_service io_service;
+    boost::asio::ip::udp::socket socket;
```

Comment:
same here, i can't see GH actions on my phone, but usually this naming is a warning. possibly not because it's public?